examples/aws/cloudformation/oss.yaml

```diff
@@ -64,18 +79,43 @@ Parameters:
 
 Mappings:
   AWSInstanceType2Arch:
+    m5.large: {Arch: HVM64}
+    m3.2xlarge: {Arch: HVM64}
+    m3.large: {Arch: HVM64}
+    m3.medium: {Arch: HVM64}
+    m3.xlarge: {Arch: HVM64}
+    m4.2xlarge: {Arch: HVM64}
+    m4.4xlarge: {Arch: HVM64}
+    m4.10xlarge: {Arch: HVM64}
+    m4.16xlarge: {Arch: HVM64}
+    m4.large: {Arch: HVM64}
+    m4.xlarge: {Arch: HVM64}
+    t2.large: {Arch: HVM64}
+    t2.medium: {Arch: HVM64}
     t2.micro: {Arch: HVM64}
+    t2.small: {Arch: HVM64}
+    t2.xlarge: {Arch: HVM64}
 
   AWSRegionArch2AMI:
     # eu-west-1 = EU (Ireland)
     # us-east-1 = US East (N. Virginia)
     # us-east-2 = US East (Ohio)
     # us-west-2 = US West (Oregon)
-    # All AMIs from AWS - gravitational-teleport-ami-oss-4.0.4
-    eu-west-1: {HVM64 : ami-077328e43d0328963}
-    us-east-1: {HVM64 : ami-02fc44456de16da53}
-    us-east-2: {HVM64 : ami-07da948a7797cc3f5}
-    us-west-2: {HVM64 : ami-0f41512a01edc93b0}
+    # All AMIs from AWS - gravitational-teleport-ami-oss-4.2.8
+    eu-west-1: {HVM64 : ami-0f5f59f75f34319cc}
+    eu-west-2: {HVM64 : ami-01e0ffba11db76e1e}
+    us-east-1: {HVM64 : ami-0beba90d621ceb1f7}
+    us-east-2: {HVM64 : ami-06efa1434fb750bac}
+    us-west-2: {HVM64 : ami-0c274d60f242b33bb}
+    us-west-1: {HVM64 : ami-05b027c18fc007d2a}
+    ap-south-1: {HVM64 : ami-05414145cf36dedd6}
+    ap-northeast-2: {HVM64 : ami-078c4f3cca7f96ffd}
+    ap-southeast-1: {HVM64 : ami-0b891741a47a6fe45}
+    ap-southeast-2: {HVM64 : ami-084e61f37e9037200}
+    ap-northeast-1: {HVM64 : ami-036435ef0149f8dc2}
+    ca-central-1: {HVM64 : ami-01aca16dbda4de6d4}
+    eu-central-1: {HVM64 : ami-0bc10cae44cb1229c}
+    sa-east-1: {HVM64 : ami-07160adda4fce8811}
```

Comment:
Good job with this. If we're going to keep offering Cloudformation templates for people, we really ought to write some scripting which will automatically go through and edit this file to have the correct AMI IDs mapped to regions whenever we publish a new version of the AMIs. I can look into doing this at some point, or if you fancy having a go at a PR then please do.